tests/CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ enable_testing()
 set(CTEST_OUTPUT_ON_FAILURE ON)
 set(library_dir "${CMAKE_SOURCE_DIR}/../libcanard")
 
+set(NO_STATIC_ANALYSIS OFF CACHE BOOL "disable canard static analysis") 
+set(NO_CANARD_SANATIZER OFF CACHE BOOL "disable canard runtime sanitizers") 
```

Comment:
Rather than stopping halfway I suggest you go forth and call it SATANIZER :imp:

Reply:
updated spelling on bdd90136ba9acd6737ea5dba821f74c379c0f865